process/sync/baseSync.go

```diff
@@ -511,7 +515,7 @@ func (boot *baseBootstrap) cleanCachesAndStorageOnRollback(header data.HeaderHan
 	boot.forkDetector.RemoveHeaders(header.GetNonce(), hash)
 	//TODO: Refactor the deletion from the headerStore as that will be an exceptional case in which,
 	//in case of "bad" rollbacks and datapool accidental eviction, the shard will halt.
-	_ = boot.headerStore.Remove(hash)
+	//_ = boot.headerStore.Remove(hash)
```

Comment:
delete commented lines. -> delete all comments, as there are not more relevant.

Reply:
deleted